document-store/src/main/java/org/hypertrace/core/documentstore/mongo/MongoCollection.java

```diff
@@ -279,7 +279,9 @@ private void parseOrderByQuery(List<OrderBy> orderBys, Map<String, Object> order
           map.put(filter.getFieldName(), value);
           break;
         case LIKE:
-          map.put(filter.getFieldName(), new BasicDBObject("$regex", value));
+          // Case insensitive regex search
+          map.put(
```

Comment:
would this leave something is searched case insensitive in a document store, and but OLAP at case sensitive?  

Does LIKE is generally case-insensitive? Or we kind need both kind of filter `LIKE` and `LIKEI`?

Reply:
OLAP is already case insensitive